spec/views/page_template_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.feature "Page Template", type: :feature do
+  scenario 'Will have a footer on home page' do
+    visit '/'
+    expect(page.source).to include("<footer>")
+    expect(page.source).to include("</footer>")
```

Comment:
I did consider making the test more sophisticated but the text, and lack of CSS, is a placeholder at present. Thought was to add another/improve existing one have styling done. It still is likely better to have gone this route however as would of highlighted change when it happens. Noted for future.